system/jlib/jstats.cpp

```diff
@@ -1236,7 +1206,18 @@ void StatisticsMapping::createMappings()
     }
 }
 
-const StatisticsMapping allStatistics;
+
+class AllStatisticsMapping : public StatisticsMapping
```

Comment:
What is the advantage in having a different mapping class?

Reply:
ok, I understand from the weird variadic constructor.

  Another alternative is to use a different constructor.  E.g., a single stat, which if it is StKindAll adds them all.

Reply:
yes considered an alternate constructor, but didn't think of a logical one to create.
Your suggestion would work though, if you prefer can change it to that.

Reply:
I think I would prefer only having a single StatisticsMapping class, so yes please if you are willing.  I don't mind what form the constructor takes (within reason!).